tests/test_xdmf.cpp

```diff
@@ -112,7 +116,11 @@ TEST( XdmfTest, Basement3HumpsTest )
     EXPECT_DOUBLE_EQ( 4.9994493491047303, max );
 
     double time = MDAL_D_time( ds );
-    EXPECT_DOUBLE_EQ( 20, time );
+    EXPECT_TRUE( compareDurationInHours( 20, time ) );
+
+    const char *referenceTime;
```

Comment:
I am thinking if you can create a function in mdal test utils hasReferenceTime( group) that returns true if reference time != 0

and in all tests just add 

EXPECT_FALSE( hasReferenceTime(g) ); 

instead of these 3-4 lines